README.md

```diff
@@ -20,24 +20,22 @@ Using the template describe your idea
 Get people excited about the idea and get them to help.
 
 ## I want to host an event
+
 ### Step 1
-Open a pull request in this repository
+Clone the repository and create a branch named after your meetup
 
 ### Step 2
-Use the template of the pull request to fill in the information needed.
+Copy the example meetup description found in [events/examples/event-title/meetup.md](events/examples/event-title/meetup.md)
```

Comment:
I think we should skip the `event-title` part of this file. Since we only have one file for the meetups now I think it makes sense to just call the `.md`-file what we want it to be named instead.

My suggestion:

`events/YYMMDD-example-event.md` is the example event
`events/YYMMDD-event-title.md` is the name of actual events (`event-title` could also be host I guess)

I kind of changed my mind about dates. In the end we want to end up with a nicely sorted list of previous events. If hosts (or us) needs to do some extra work like renaming the event before merging the PR I think it's worth it. Also, this is easier when it's not a folder but instead just a single file.